.github/terraform-docs/module.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright 2022 Google LLC
```

Comment:
thanks for getting this ready @Shabirmean! 
Quick question : for the terraform-docs, i see that there is both a `module.yaml` and `main.yaml` in the .github folder. Is it possible to combine them, or do they need to be separated?

Reply:
oh nevermind, i see the flow!